images/Dockerfile.rhel

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14 AS builder
```

Comment:
maybe also update the runner image?

Reply:
You can't change the base image to RHEL 9 without also changing the build image to RHEL 9, because RHEL 8 and 9 have different versions of OpenSSL. This change has broken all FIPS clusters.